metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -541,30 +528,7 @@ def create(
         )
         obj.echo(f"\nDone writing *{current.flow_name}* {kind} to {pipeline_path}")
     else:
-        obj.echo(f"Deploying *{flow.name}* to Argo Workflows...", bold=True)
-        workflow_template = flow.deploy(
-            kubernetes_namespace,
-            workflow_name,
-            flow_parameters=flow_parameters,
-            recurring_run_enable=recurring_run_enable,
-            recurring_run_cron=recurring_run_cron,
-            recurring_run_policy=recurring_run_concurrency,
-            max_run_concurrency=max_run_concurrency,
-        )
-        template_name = workflow_template["metadata"]["name"]
-        obj.echo(
-            f"Workflow Template *{template_name}* "
-            f"for flow *{current.flow_name}* pushed to "
-            "Argo Workflows successfully.\n",
-            bold=True,
-        )
-        argo_ui_url = f"{ARGO_RUN_URL_PREFIX}/argo-ui/workflow-templates/{kubernetes_namespace}/{template_name}"
-        obj.echo(f"*Argo:* {argo_ui_url}", fg="cyan")
-
-        # TODO(talebz): using @schedule decorator?
-        #  flow.schedule()
-        #  obj.echo("What will trigger execution of the workflow:", bold=True)
-        #  obj.echo(flow.trigger_explanation(), indent=True)
+        raise KfpException("create command is only supported with --yaml-only")
```

Comment:
Does it make sense to swap the default of `yaml_only` to be `True` now to make this the happy path?

Reply:
The concern is what happens when this command would truly allow creating in the cluster?
So making it explicit that to be yaml_only is the approach.  Furthermore, this is an advanced command